hw/top_earlgrey/ip/xbar_peri/rtl/autogen/xbar_peri.core

```diff
@@ -0,0 +1,23 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# xbar_peri core file generated by `tlgen.py` tool
+name: "lowrisc:ip:xbar_peri:0.1"
+description: "Generated RTL xbar_peri"
+
+filesets:
+  files_rtl:
+    depend:
+      - lowrisc:ip:tlul
+    files:
+      - tl_peri_pkg.sv
+      - xbar_peri.sv
+    file_type: systemVerilogSource
+
+targets:
+  default: &default_target
+    filesets:
+      - files_rtl
+    toplevel: xbar_main
```

Comment:
I think it doesnt matter, but make this `xbar_peri` for correctness.

Reply:
Good catch, thanks. Updated